src/index.ts

```diff
@@ -0,0 +1,10 @@
+import injectStylesheet from "./lib/inject-stylesheet";
+
+export = {
+  injectWithAllowlist: function (styles, list): HTMLStyleElement {
+    return injectStylesheet(styles, list, true);
+  },
```

Comment:
Using arrow functions here will compile down to functions (nonbreaking change that is) bc our target is ES5.

```suggestion
  injectWithAllowlist: (styles, list): HTMLStyleElement =>
    injectStylesheet(styles, list, true),
```

Reply:
Opted to go with the normal export here instead since we can do it without changing how the shape of the commonjs module looks: https://github.com/braintree/inject-stylesheet/pull/10/commits/df0c437edc440e2422782a22e0005775746c0284